openvpn-monitor.py

```diff
@@ -25,6 +25,11 @@
 import socket
 import re
 import argparse
+try:
+    import geoip2.database
+except ImportError:
+    # No geoip2 support
+    pass
```

Comment:
Maybe import GeoIP instead of `pass`? And set `use_geopi2`? Or maybe change that variable to be called `geoip_version`?

Reply:
I'm not sure whether geoip2 lib can read geoip1 databases, i'm afraid that in order to keep backwards compatibility both are needed.
I'll try to make it look like less of an hack anyway.

What about a wrapper arround geoip1/2 on a separate file?

Reply:
Maybe if the db filename ends in mmdb, try to import geoip2? Otherwise try to import geoip?

Or maybe an option in the config file -> `geoip_version = 1` and just import whichever version is specified.